doc/doxygen/src/porting-boards.md

```diff
@@ -135,11 +135,8 @@ e.g.:
 PORT_LINUX ?= /dev/ttyUSB0
 PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbserial*)))
```

Comment:
I would add a comment that this is only if you need to override defaults

Reply:
I extended the documentation.